core/GlobalState.cc

```diff
@@ -58,11 +58,20 @@ GlobalState::GlobalState(shared_ptr<ErrorQueue> errorQueue, shared_ptr<lsp::Type
       lspQuery(lsp::Query::noQuery()), epochManager(move(epochManager)) {
     // Empirically determined to be the smallest powers of two larger than the
     // values required by the payload
-    unsigned int maxNameCount = 8192;
-    unsigned int maxSymbolCount = 16384;
+    unsigned int maxNameCount = 32768;
```

Comment:
Why did these grow?
both in sum and each separately?


Is this to fit current payload?

Reply:
(maybe let's make this into a test?)

Reply:
To fit current payload.

Reply:
I added an ENFORCE in payload that will fail in tests.